### PR TITLE
Challenge 10 greghilston new

### PR DESCRIPTION
**Pull request by GregHilston**
This is my very first program that involves HTMl, CSS and JS. I would very much appreciate any feedback on my `script.js`.

I did not complete the optional bonus portion of the challenge. I originally did not have global variables, but couldn't get `setInterval` to be called with a variable, so I gave up.

See: https://repl.it/@GregHilston/Challenge10MakingALoadingBar

Thank you!

_Note: I am not the author of the `html` or `css`, but would openly accept and feedback on it!_

### CODE REVIEW

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <label for="secondsinput">Load Time (in seconds)</label>
+    <input id="secondsinput" type="number" min="0" value="3"><br>
```
Comment (theRingleman) [inline Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html]: Typically people don't use `<br>` anymore. The recommended way is strictly use CSS to do any kind of visual changes. Makes your markup a lot cleaner and easier to figure out what is causing that blank line.

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/style.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#loadingbar {
+  height:30px;
```
Comment (theRingleman) [inline Challenges/challenge_10_making_a_loading_bar/greghilston/src/style.css]: Stylistic nitpick: typically you would insert a space after the `:`. e.g.
```
#loadingbar {
  height: 30px;
}```

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+var totalTime;
+var curTime;
+
+var setButton = document.getElementById("setload");
+var loadingBar = document.getElementById("progress");
```
Comment (theRingleman) [inline Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js]: I am pretty sure this could be a const instead of var.

Comment (theRingleman) [reply Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js]: Actually both of these could be.

Comment (GregHilston) [reply Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js]: Oh good catch!

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html (hunk 1 above)
Comment (adamwhitlock1) [reply Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html]: Additionally I believe that the <br> is technically supposed to be written as a self closing tag like:
```
<br />
```
I would agree with @theRingleman in that <br /> element should not generally be used for presentational purposes, unless it is added inside a content block to create a line wrap for specific content types (think poems, addresses, etc)

Something this example:

```
<p>
    Boaty mcBoatface
    20 street name<br />
    City Name<br />
    etc.
</p>
```
That being said, I am the first to admit that I have used <br /> far too often during quick edits on sites, where I should have used a span and css.

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/script.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+const oneSecondMiliSeconds = 1000
+let progressBar = document.getElementById("progress")
```
Comment (kkroboth) [inline Challenges/challenge_10_making_a_loading_bar/greghilston/src/script.js]: nit:
```suggestion
const progressBar = document.getElementById("progress")
const resetLoadingBarButton = document.getElementById("setload")
```
It's a good practice to always use `const` unless they will be re-assigned.

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html (hunk 1 above)
Comment (kkroboth) [inline Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html]: I did the same mistake too, but `input`  props `min`, and `max` are not strictly enforced. I haven't used these properties much, but I'm sure it's only for form submissions. For example, if you input `-5`, the browser won't allow a form `HTTP POST` submission.

File: Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js (hunk 3 above)
Comment (kkroboth) [reply Challenges/challenge_10_making_a_loading_bar/greghilston/src/theanswer.js]: Yes, and I would stay away from `var`. The `var` declaration is scoped to it's nearest function (in this case it's the global context `window`). `let` and `const` are scoped to their code blocks, e.g., if statement, for loop, `{}`.

**Comment by kkroboth** [comment]
LGTM 🚢! Some nits on constants and input validation.

**Pull request merged**